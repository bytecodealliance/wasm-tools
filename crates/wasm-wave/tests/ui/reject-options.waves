option-u8(some);
option-u8(some());
option-u8(some(some));
option-u8(some(some()));
option-u8(some(some(0)));
option-u8(some(some(some(0))));
option-u8(none());
option-u8(some(none()));
option-u8(none(0));
option-u8(some(none(0)));
option-u8(none(some(0)));