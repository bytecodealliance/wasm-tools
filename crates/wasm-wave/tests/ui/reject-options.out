unexpected token: ParenClose at 14..15
unexpected token: ParenClose at 15..16
unexpected token: ParenClose at 19..20
unexpected token: ParenClose at 20..21
invalid value type at 15..22
invalid value type at 15..28
unexpected token: ParenOpen at 14..15
unexpected token: ParenOpen at 19..20
unexpected token: ParenOpen at 14..15
unexpected token: ParenOpen at 19..20
unexpected token: ParenOpen at 14..15
